lib/rules/sotd/charter.js

```diff
@@ -7,18 +7,30 @@ const self = {
 
 exports.name = self.name;
 
+const util = require('../../util');
+
 const charterText =
     /The disclosure obligations of the Participants of this group are described in the charter\./;
 
 exports.check = async function (sr, done) {
     const sotd = sr.getSotDSection();
     if (sotd) {
-        const deliverId = await sr.getDelivererIDs();
-        if (!deliverId.length) {
+        const deliverIds = await sr.getDelivererIDs();
+
+        if (!deliverIds.length) {
             sr.error(self, 'no-group');
             return done();
         }
 
+        // Skip check if the document is only published by TAG and/or AB
+        const TagID = util.TAG.id;
+        const AbID = util.AB.id;
+        // groupIds: a list of ids without TAG or AB
+        const groupIds = deliverIds.filter(
+            deliverer => [TagID, AbID].indexOf(deliverer) < 0
+        );
+        if (!groupIds.length) return done();
```

Comment:
As above, we should consider the case of a joint publication between the TAG (or AB) and a WG